### PR TITLE
Update nlohmann_json to tag 3.11.3 for benchmark_app

### PR DESCRIPTION
**Pull request by peterchen-intel**

### CODE REVIEW

**Comment by peterchen-intel** [comment]
Need to check if samples tests are triggered.

File: thirdparty/json/nlohmann_json
Comment (Wovchena) [inline thirdparty/json/nlohmann_json:1]: What's the reason for the update?

Comment (wenjiew) [reply thirdparty/json/nlohmann_json:1]: There are two critical security issues discovered in the dependency of 3.11.2 version, which is required be upgraded according to the SDL side.

**Review by Wovchena — APPROVED** [review]

**Comment by peterchen-intel** [comment]
> Need to check if samples tests are triggered.

e2e-tests-linux-ubuntu20-samples was triggered and passed.

**Pull request merged**